client/src/partials/enterprises/enterprises.html

```diff
@@ -20,235 +20,181 @@
-                validation-trigger="UpdateForm.$submitted"
+                validation-trigger="EnterpriseForm.$submitted"
                 location-uuid="EnterpriseCtrl.enterprise.location_id">
               </bh-location-select>
 
-              <div class="form-group" ng-class="{ 'has-error' : UpdateForm.$submitted && UpdateForm.currency_id.$invalid }">
-                <p><strong class="control-label">{{ "FORM.LABELS.CURRENCY" | translate }}</strong></p>
-                <div class="radio" ng-repeat="currency in EnterpriseCtrl.currencies track by currency.id">
-                  <label class="control-label">
-                    <input type="radio" value="{{currency.id}}"  id="currency_id-{{currency.id}}" name="currency_id" ng-model="EnterpriseCtrl.enterprise.currency_id" required>
-                    {{ currency.name }}
-                  </label>
-                </div>
-                <div class="help-block" ng-messages="UpdateForm.currency_id.$error" ng-show="UpdateForm.$submitted">
-                  <div ng-messages-include="partials/templates/messages.tmpl.html"></div>
-                </div>
-              </div>
 
-              <div class="form-group">
-                <bh-loading-button loading-state="UpdateForm.$loading">
-                  {{ "FORM.BUTTONS.SAVE" | translate }}
-                </bh-loading-button>
-                <button class="btn btn-default" type="button" ng-click="EnterpriseCtrl.cancel()" data-method="cancel">
-                  {{ "FORM.BUTTONS.RESET" | translate }}
-                </button>
-              </div>
-            </form>
+              <button class="btn btn-sm btn-block btn-default" type="button" ng-click="EnterpriseCtrl.submit(EnterpriseForm)" data-method="cancel">
+                <i class="fa fa-save"></i> {{ "FORM.BUTTONS.UPDATE" | translate }}
+              </button>
+            </div>
+
           </div>
-        </div>
+
+        </form>
+      </div>
+
+      <!-- project form  -->
+      <div class="col-xs-12 col-sm-6">
+        <form
+          name="ProjectForm" novalidate>
+
+          <div class="panel panel-primary">
+            <div class="panel-heading">
+              <b><i class="fa fa-cube"></i> {{ 'TREE.PROJECT' | translate }}</b>
+            </div>
+
+            <ul class="list-group">
+              <li class="list-group-item" ng-repeat="p in EnterpriseCtrl.projects">
+                <i class="fa fa-hospital-o"></i>
+                <span>{{ p.name }} ({{ p.abbr }})</span>
+                <a data-delete="{{ p.abbr }}" class="badge badge-danger" ng-click="EnterpriseCtrl.deleteProject(p.id, p.name)"><i class="fa fa-trash"></i> {{ 'FORM.BUTTONS.DELETE' | translate }}</a>
+                <a data-update="{{ p.abbr }}" class="badge badge-default" ng-click="EnterpriseCtrl.editProject(p.id)"><i class="fa fa-edit"></i> {{ 'FORM.BUTTONS.EDIT' | translate }}</a>
+              </li>
+            </ul>
+
+            <div class="panel-footer clearfix">
```

Comment:
Instead of 
```html
<div class="panel-footer clearfix">
  <button class="pull-right">
    ....
```
use
```html
<div class="panel-footer text-right">
  <button>
     ....
```
Clearfix is considered a hack by most web developers since it relies on pseudoelements.